src/script/api/script_companymode.hpp

```diff
@@ -47,6 +47,24 @@ class ScriptCompanyMode : public ScriptObject {
 	 *   in when the instance was created.
 	 */
 	~ScriptCompanyMode();
+
+	/**
+	 * Check whether a company mode is valid. In other words, are commands
+	 * being executed under some company.
+	 * @return true When a company mode is valid.
+	 * @post !ScriptCompanyMode::IsDeity().
+	 * @api gs
```

Comment:
it's `@api game` IIRC

Reply:
Well... doxygen_filter.awk:23 changes gs to game, so it should function. But the whole `ScriptCompanyMode` is already only for the game script, so it's not even needed at all.